source/_templates/installations/elastic/common/generate_elastic_credentials.rst

```diff
@@ -4,7 +4,7 @@
 
   # /usr/share/elasticsearch/bin/elasticsearch-setup-passwords auto
 
-The ``elastic``'s password must be saved.
+The ``elastic's`` password must be saved.
```

Comment:
Not sure about that. The user is `elastic`, so de `'s` need to be out of the format.